vms/shift/tests/test_manageVolunteerShift.py

```diff
@@ -480,6 +484,42 @@ def test_cancel_assigned_shift(self):
         self.assertEqual(slots_remaining_before_assignment,
                          slots_after_cancellation)
 
+    def test_shift_edit_request(self):
+        volunteer_1 = create_volunteer_with_details(self.volunteer_1)
+
+        shift = ['09:00', '15:00', '1']
+        shift_1 = self.create_shift(shift)
+        start = datetime.time(hour=10, minute=0)
+        end = datetime.time(hour=14, minute=0)
+        logged_shift = log_hours_with_details(volunteer_1, shift_1, start, end)
+        start_time = datetime.time(hour=9, minute=30)
+        end_time = datetime.time(hour=14, minute=0)
+        edit_request = create_edit_request_with_details(start_time, end_time, logged_shift)
+        shift_id = shift_1.id
```

Comment:
if we don't have any further use of these vars I think we can use it directly, views?